Firebase/Core/Private/FIRAppInternal.h

```diff
@@ -83,6 +83,13 @@ extern NSString *const FIRAuthStateDidChangeInternalNotificationTokenKey;
  */
 extern NSString *const FIRAuthStateDidChangeInternalNotificationAppKey;
 
+/** @var FIRAuthStateDidChangeInternalNotificationUIDKey
+ @brief A key present in the dictionary object parameter of the
+ @c FIRAuthStateDidChangeInternalNotification notification. The value associated with this
+ key will contain the new user's UID.
```

Comment:
To be thorough perhaps add "(or nil if there is no longer a user signed in)"

Reply:
(but on the upstream PR #507 that implements this.)

Reply:
Done (on upstream PR.) I'll rebase this PR shortly after which the changes will also be visible here.